crates/subspace-node/src/domain/evm_chain_spec.rs

```diff
@@ -318,8 +275,8 @@ fn testnet_genesis(
         sudo: SudoConfig {
             key: maybe_sudo_account,
         },
-        transaction_payment: Default::default(),
         balances: BalancesConfig {
+            // TODO: remove `endowed_accounts` once XDM is ready
```

Comment:
Because the genesis storage is registered with the runtime code and is reused by every domain instance, so it basically means we mint SSC whenever a domain is instantiated.

Reply:
sure, for the production runtime these will be empty anyway. For the tests, I would prefer having these endowed accounts rather than using the XDM to moving funds from the Consensus. This is a bad ux when testing or developing

Reply:
Okay, will update the comment